Makefile

```diff
@@ -36,6 +36,7 @@ $(services): target/spec $(openapi-generator-jar)
 		-t templates/csharp \
 		-o $(output) \
 		--model-package $@ \
+		--skip-validate-spec \
```

Comment:
why do we add this flag?

Reply:
The generator performs a validation on the specs it tries to build on. Since our specs are a bit wonky, it would throw errors and not complete generation. Hence the this flag :)